src/simulate_drone_services/all_drones_available_service.py

```diff
@@ -0,0 +1,30 @@
+'''
+Since we don't have the physical drone to advertise its services (land, takeoff, etc)
+this script simulates advertising those services. This lets the server actually call them.
+Instead of seeing the perform its action, we will get a print message.
+'''
+
+import roslibpy
+import json
+
+client = roslibpy.Ros(host='54.161.15.175', port=9090)
```

Comment:
Change ip address to refer to constants file.  Do this for all files in simulate_drone_services and in roslibpy_test.